cfme/tests/containers/test_projects_dashboard.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+
+from cfme.containers.provider import ContainersProvider
+from cfme.containers.container import Container
+from cfme.containers.image import Image
+from cfme.containers.service import Service
+
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(lambda provider: current_version() < "5.9"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1),
+    pytest.mark.provider([ContainersProvider], scope='function')
+]
+
+TESTED_OBJECTS = [Container, Image, Service]
+PROJECT_NAME = 'test-project-dashboard'  # Predefine project for this test
+
+
+def get_api_object_counts(appliance, project_name, provider):
+    """ Fetches amount of Containers/Services/Images from the API per selected project name"""
+    assert isinstance(provider, ContainersProvider)
+    return {
+        Container: len(provider.mgmt.list_container(project_name=project_name)),
+        Service: len(provider.mgmt.list_service(project_name=project_name)),
+        Image: len(get_container_images_amt(provider, project_name))
+    }
+
+
+def get_container_images_amt(provider, project_name=None):
+    """ Fetches images amount from the API per selected project name"""
+    images = provider.mgmt.list_image()
+    project_images = []
+    for image in images:
+        if image.image_project_name == project_name:
+            project_images.append(image)
+    return project_images
+
+
+def get_api_pods_names(provider):
+    """ Fetches Pod names from the API per selected project name"""
+    pod_name = []
+    for pod in provider.mgmt.list_container_group(project_name=PROJECT_NAME):
+        pod_name.append(pod.name)
+    return pod_name
+
+
+@pytest.mark.polarion('CMP-10806')
+def test_projects_dashboard_pods(provider, appliance, soft_assert):
+    """Tests data integrity of Pods names in Pods status box in Projects Dashboard.
+    Steps:
+        * Go to Projects / Dashboard View
+        * Compare the data in the Pods status box to API data for
+        Pods names
+    """
+    api_pod_names = get_api_pods_names(provider)
+    collection_object = appliance.collections.container_projects
+    inst = collection_object.instantiate(name=PROJECT_NAME, provider=provider)
```

Comment:
**Required**

Use a fixture to generate the project instance, eliminating duplicate code and moving this functionality into test setup.

Reply:
Create new fixture container_project_instance, use it instead of duplication code.